OpenRA.Game/Traits/Player/FrozenActorLayer.cs

```diff
@@ -211,5 +220,14 @@ public FrozenActor FromID(uint id)
 
 			return ret;
 		}
+
+		public IEnumerable<Pair<CPos, Color>> RadarSignatureCells(Actor self)
+		{
+			// Filter only actors that have been discovered by the player
+			// HasRenderables is only True once the actor enters the viewport, so is not enough for the minimap
+			foreach (var frozenActor in frozen.Values.Where(f => f.NeedRenderables || f.HasRenderables))
```

Comment:
Maybe we should rename ```NeedsRenderables``` to ```HasBeenSeen``` or something like that, since it's not strictly about renderables any more.